src/goose/specs.clj

```diff
@@ -108,8 +108,10 @@
 
 ;;; ============== Batch ==============
 (s/def ::callback-fn-sym (s/nilable ::fn-sym))
+(s/def ::linger-in-hours nat-int?)
 (s/def ::batch-opts
-  (s/keys :opt-un [::callback-fn-sym]))
+  (s/keys :req-un [::linger-in-hours]
```

Comment:
suggestion: why required? Can we not default this to 24hrs. I think thats not setup yet, plz add that

Reply:
@olttwa had recommended we expect the user to pass in default args. I think this is by convention (on how Goose opts are configured).

I also think we should consider moving it to Goose defaults.

Reply:
Added default linger and moved `linger-in-hours` to optional in spec.